lib/fog/compute/google/models/instance_groups.rb

```diff
@@ -32,17 +32,15 @@ def get(identity, zone = nil)
           nil
         end
 
+        # TODO: This shouldn't be in this model, add deprecation warning
         def add_instance(params)
```

Comment:
I think you can use `Fog` api to add this deprecation. Is `Fog.deprecate` if i'm not mistaken.